Random NullRef exception when clicking left menu items

```
00:00:41.975 [ERROR] Encountered an exception published to an object with an unobserved ThrownExceptions property|System.NullReferenceException: Object reference not set to an instance of an object.
   at NexusMods.App.UI.WorkspaceSystem.WorkspaceController.GetOpenPageBehavior(PageData requestedPage, NavigationInformation navigationInformation) in C:\Dev\Repos\NexusMods.App\src\NexusMods.App.UI\WorkspaceSystem\WorkspaceController\WorkspaceController.cs:line 408
   at NexusMods.App.UI.LeftMenu.Items.LeftMenuItemViewModel.<>c__DisplayClass27_0.<.ctor>b__0(NavigationInformation info) in C:\Dev\Repos\NexusMods.App\src\NexusMods.App.UI\LeftMenu\Items\LeftMenuItemViewModel.cs:line 40
   at ReactiveUI.ReactiveCommand.<>c__DisplayClass4_1`1.<Create>b__1(IObserver`1 observer) in c:\temp\releaser\ReactiveUI\src\ReactiveUI\ReactiveCommand\ReactiveCommand.cs:line 193
   at System.Reactive.Linq.QueryLanguage.CreateWithDisposableObservable`1.SubscribeCore(IObserver`1 observer)
   at System.Reactive.ObservableBase`1.Subscribe(IObserver`1 observer)
```
I got this a couple of times when clicking on a left menu item, but I haven't been able to reproduce since. 

So repro steps are unclear for now

--- a/src/NexusMods.App.UI/Controls/Navigation/NavigationControl.cs
+++ b/src/NexusMods.App.UI/Controls/Navigation/NavigationControl.cs
@@ -90,8 +90,8 @@ public class NavigationControl : StandardButton
     {
         // NOTE(erri120): Can't get the input in this event, we'll just re-use
         // the value set in OnPointerPressed.
-        // NOTE(Al12rs): It's possible that OnPointerPressed was not called.
-        // Happens very rarely, not sure how to reproduce, but we should just ignore it if it happens.
+        // NOTE(Al12rs): It's somehow possible that OnPointerPressed was not called before OnPointerReleased.
+        // Happens very rarely, not sure how or why, but we should just ignore it if it happens.
         Debug.Assert(_wasOnPointerPressedCalled);
         if (!_wasOnPointerPressedCalled) return;
         _wasOnPointerPressedCalled = false;

--- a/src/NexusMods.App.UI/Controls/Navigation/NavigationControl.cs
+++ b/src/NexusMods.App.UI/Controls/Navigation/NavigationControl.cs
@@ -89,9 +89,11 @@ public class NavigationControl : StandardButton
     protected override void OnPointerReleased(PointerReleasedEventArgs e)
     {
         // NOTE(erri120): Can't get the input in this event, we'll just re-use
-        // the value set in OnPointerPressed. This assumes that OnPointerPressed
-        // always gets called before OnPointerReleased.
+        // the value set in OnPointerPressed.
+        // NOTE(Al12rs): It's possible that OnPointerPressed was not called.
+        // Happens very rarely, not sure how to reproduce, but we should just ignore it if it happens.
         Debug.Assert(_wasOnPointerPressedCalled);
+        if (!_wasOnPointerPressedCalled) return;
         _wasOnPointerPressedCalled = false;
 
         base.OnPointerReleased(e);

